### ISSUE TITLE
[photospline] convolve.c missing from package-test branch (Trac #1729)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1721]
For the standalone package-test branch: splinetable_convolve() is referenced in the docs and in the headers, but convolve.c is not included.  

Migrated from https://code.icecube.wisc.edu/ticket/1729
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "\nFor the standalone package-test branch: splinetable_convolve() is referenced in the docs and in the headers, but convolve.c is not included.  \n\n", 
    "reporter": "jkelley", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "combo reconstruction", 
    "summary": "[photospline] convolve.c missing from package-test branch", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-06-06T19:50:29", 
    "milestone": "", 
    "owner": "jkelley", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160606T20:04:35: 
* jkelley commented:

> In changeset 146911:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="146911"
> Add convolve.c from trunk, include in CMakeLists sources.  Fixes #1721.
> ```


* jkelley changed status from "new" to "closed"
* jkelley changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
